data/simpy.gram

```diff
@@ -324,10 +324,28 @@ power[expr_ty]:
     | primary 
 primary[expr_ty]:
```

Comment:
I wonder if we could rename this to `await_primary` and then rename `primary_raw` to just `primary`.